app/models/user.rb

```diff
@@ -101,9 +101,12 @@ def with_team_kind(kind)
     end
 
     def with_all_associations_joined
-      includes(:conferences).group("conferences.id").
-      includes(:roles).group("roles.id").
-      includes(roles: :team).group("teams.id")
+      eager_load(:conferences).
```

Comment:
Thanks Yorick! 
I understood, there are 2 ways of `preloading`; `preload` and `eager load`. You can force `eager load` yourself, or if you do `include`, you let Rails decide which one to use. That's where I stopped investigating.

What I still don't understand, is why the `.group()` clauses are needed with `include`, and are not in the `eager_load`s. 
But now that we are talking about this, I come to think that probably André expected ME to add those clauses. Check: @andremedeiros :that right? Because, as I know now, they are SQL and have nothing to do with loading. Right?